Geolocator/Geolocator/Geolocator.Plugin.iOS/GeolocatorImplementation.cs

```diff
@@ -216,8 +218,10 @@ public Task<Position> GetPositionAsync(int timeoutMilliseconds = Timeout.Infinit
             return tcs.Task;
         }
         /// <inheritdoc/>
-        public Task<bool> StartListeningAsync(int minTime, double minDistance, bool includeHeading = false)
+		public Task<bool> StartListeningAsync(int minTime, double minDistance, bool includeHeading = false, EnergySettings energySettings = null)
         {
+			this.energySettings = energySettings;
```

Comment:
Right here we should do a check against iOS 6 and if it is < iOS 6 never set it so we never have to check again.